RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -627,15 +627,17 @@ boolean L12_getOutfit()
 	if(auto_warSide() == "fratboy" && possessOutfit("Filthy Hippy Disguise"))
 	{
 		autoOutfit("Filthy Hippy Disguise");
-		//r25671 this should be [Frat House In Disguise] but due to mafia issue it currently needs to be as below
+		//r25671 this should be [Frat House (Frat Disguise)] but due to mafia issue it currently needs to be as below
+		//r26631 "In Disguise" location names changed; this issue not checked
 		return autoAdv($location[Frat House]);
```

Comment:
`$location[Frat House (Frat Disguise)]` correctly returns Frat House (Frat Disguise) now, so this can be updated to the desired location now.

```suggestion
		return autoAdv($location[Frat House (Frat Disguise)]);
```